tools/crates-validator/src/gen_profiles/mod.rs

```diff
@@ -14,7 +14,6 @@
 
 pub mod args;
 
-// TODO(Cardosaum): Check how to export only the functions that are needed
```

Comment:
Curious where the exported types here are used, I don't see them in this repo?

Reply:
They're only used [here](https://github.com/risc0/risc0/blob/1005dca1e9943810a9523e56e47bbd5c696505f6/tools/crates-validator/src/bin/gen-profiles.rs#L29). 

The types need to be public since the methods are and they return those intermediary types. Not sure if this would be the best approach, feel free to recommend refactoring if you feel appropriate!